### ISSUE TITLE
[Bug]: Text styles of bold/italic/underline are not saved in history

### ISSUE THREAD

**Issue by adamkoch** [hm21/pro_image_editor#118]
### Package Version

4.0.4

### Flutter Version

3.22.2

### Platforms

iOS

### How to reproduce?

I was able to create my own bold/italic/underline buttons for the text editor as per your recommendation here:
https://github.com/hm21/pro_image_editor/issues/67

It all works fine and the saved image has the styles (with the exception of bold which still doesn't seem to take effect, see comment here: https://github.com/hm21/pro_image_editor/issues/67#issuecomment-2156307329).

However, the history does not seem to save these parts of the text style. When I reload the history the text has resorted back to the plain style with no underline/italic.

To set underline, I use an AppBar icon that triggers this:
```
final newDecoration = textEditor.selectedTextStyle.decoration == TextDecoration.underline
    ? TextDecoration.none
    : TextDecoration.underline;
final newStyle = textEditor.selectedTextStyle.copyWith(decoration: newDecoration);
textEditor.setTextStyle(newStyle);
```

**Comment by hm21** [comment]
Thanks for reporting this issue. 

To be honest, I didn't see your comment in #67 and thought it's resolved already😅

Anyway, I just released version `4.0.5` which should fix it. Please let me know if this issue still exists by you.

**Comment by adamkoch** [comment]
Thanks, fixed!!
